Mixpanel/Mixpanel.m

```diff
@@ -693,6 +702,31 @@ - (NSURLRequest *)apiRequestWithEndpoint:(NSString *)endpoint andBody:(NSString
     return request;
 }
 
+- (void)handleNetworkResponse:(NSHTTPURLResponse *)response withError:(NSError *)error
+{
+    NSNumber *retryTime = response.allHeaderFields[@"Retry-After"];
```

Comment:
We're not using the header information at all. If this field is present we should pick this time if it's larger than `pow(2.0, failureCount) * 60 + arc4random_uniform(30)`.